src/Libraries/DesignScriptBuiltin/RangeHelpers.cs

```diff
@@ -31,7 +31,7 @@ public enum RangeStepOperator
 namespace Builtin
 {
     //TODO for now suppress import into DS VM.
-    [SupressImportIntoVM]
+    [IsVisibleInDynamoLibrary(false)]
```

Comment:
@mjkkirschner I exposed these to the VM as a shortcut to getting access to these types from the same place (the old vm's type system)